### ISSUE TITLE
[CLI] autocompletion doesn't suggest directories

### ISSUE THREAD

**Issue by daro1337** [scramjetorg/transform-hub#397]
**Describe the bug**
autocompletion doesn't suggest files and directories e.g in `si pack`

**Steps to Reproduce**
1. type `si pack  ` and press tab for autocompletion

**Expected behavior**
`si pack` autocoplete files and directories to conveniently indicate a path with a sequence

**Additional information like logs, screenshots etc.**
-

**Comment by alicja-gruzdz** [comment]
Indeed, agree, only I'm not sure if it's a bug or a feature :) because I think the original CLI autocomplete task was for CLI commands only - right, @kociolekscramjet ? 
But I see would be useful to have files/dirs for `pack`. I'm not sure what is the time effort, maybe @kociolekscramjet can comment on it. He did a marvelous job on autocomplete but I remember it wasn't super easy to make it work.
One thing I would change in the future in CLI autocomplete script is moving it to a separate file and only including it in .bashrc., because it is making .bashrc messy. Plus, I think it installed the script twice when I accidentally ran the install command again:

![Zrzut ekranu z 2022-04-05 10-15-49](https://user-images.githubusercontent.com/8177865/161709937-52fbf19e-03eb-48d4-831e-c074ee997650.png)

**Comment by kociolekscramjet** [comment]
Yes as you mentioned @alicja-gruzdz initial task mentioned autocompletion for command names only. But those are good suggestions! As the initial part is completed, and is working now, both changes - the filename completion and the failsafe cleaner way of injection into bashrc shouldn't be too time consuming.

The filename completion: I think it should be covered with modification in commands/completion.ts:25 where proper data is provided but we need to write the handler for the completion - a little research needed

The cleaner installation script: Changes should be done in here commands/completion.ts:47 for now it's just si completion bash script piped to users bashrc.
